packages/ui-components/src/components/DataGridRow/DataGridRow.component.tsx

```diff
@@ -4,32 +4,14 @@
  */
 
 import React, { forwardRef } from "react"
+import "./data-grid-row.css"
 
-const rowBaseStyle = `
-	jn:contents
-`
-// const rowSelectedStyle = `
-// 	jn:bg-theme-datagridrow-selected
-// `
 export const DataGridRow = forwardRef<HTMLDivElement, DataGridRowProps>(
-  ({ /* selected, disabled,*/ className = "", children, /*onChange,*/ ...props }, ref) => {
-    // const dataGridContext = useDataGridContext() || {}
-    // const selectable = dataGridContext.selectable
+  ({ hoverable = false, className = "", children, ...props }, ref) => {
+    const finalStyles = `${hoverable ? "juno-datagrid-group" : ""} juno-datagrid-row ${className}`
```

Comment:
The name `juno-datagrid-group` doesn't suggest that this is what adds the `hoverable` behaviour. We try to give semantic names where possible. Here my suggestion is to use `juno-datagrid-row-hoverable` instead. Like this:
```suggestion
    const finalStyles = `juno-datagrid-row ${hoverable ? "juno-datagrid-row-hoverable" : ""} ${className}`
```

Reply:
Also please keep the classname string where it was with the `<div>`. It's not that complex yet that it requires being offloaded into a separate const and makes it easier to read.